Source/Diagnostics/Diagnostics.H

```diff
@@ -104,6 +106,21 @@ public:
     void FilterComputePackFlush (int step, bool force_flush=false);
     /** Whether the last timestep is always dumped */
     bool DoDumpLastTimestep () const {return  m_dump_last_timestep;}
+    int getnumbuffers() {return m_num_buffers;}
```

Comment:
Since this is the Diagnostics general interface class, you add docstrings here? :)

Reply:
yes. thats what i have done now :)